controller/controller_server/utils.py

```diff
@@ -158,6 +158,29 @@ def validate_delete_snapshot_request(request):
     logger.debug("request validation finished.")
 
 
+def validate_expand_volume_request(request):
+    logger.debug("validating expand volume request")
+
+    if not request.volume_id:
+        raise ValidationException(messages.name_should_not_be_empty_message)
+
+    logger.debug("validating volume capacity")
+    if request.capacity_range:
+        if request.capacity_range.required_bytes < 0:
+            raise ValidationException(messages.size_bigger_than_0_message)
+    else:
+        raise ValidationException(messages.no_capacity_range_message)
+
+    validate_secret(request.secrets)
+
+    logger.debug("validating volume capabilities")
+    if not request.volume_capability:
+        raise ValidationException(messages.capabilities_not_set_message)
```

Comment:
I thought `This is an OPTIONAL field.`

Reply:
removed it, didnt used capabilities

Reply:
ok, please note that it is still required for this feature, just easier to be done in a [separate PR](https://github.com/IBM/ibm-block-csi-driver/pull/212#pullrequestreview-528927603)